Add initial guidelines and checklists (#1)
sandstreamdev/sandcoding: guidelines/github-pages.md

# Github pages

> If you have some example application that is only frontend app then you can use `Github pages` to make it work online.

Reviewer: ```suggestion
If you have some example application that is a frontend-only app then you can use [GitHub Pages](https://pages.github.com/) to make it work online.
```

> Us following [project](https://github.com/tschaub/gh-pages) to just deploy with one script.

Reviewer: Use?

```suggestion
Use the following module [project](https://github.com/tschaub/gh-pages) to deploy with a single script call.
```

> Us the following [module](https://github.com/tschaub/gh-pages) to deploy with a single script call.

Reviewer: ```suggestion
Use the following [module](https://github.com/tschaub/gh-pages) to deploy with a single script call.
```
